system/src/Grav/Common/GPM/GPM.php

```diff
@@ -139,6 +139,32 @@ public function getInstalledPlugins()
         return $this->installed['plugins'];
     }
 
+
+    /**
+     * Returns the plugin's enabled state
+     *
+     * @param  string $slug
+     * @return bool True if the Plugin is Enabled. False if manually set to enable:false. Null otherwise.
+     */
+    public function isPluginEnabled($slug)
+    {
+        $result = null;
+        $grav = new Grav();
```

Comment:
This won't work. Use `$grav = Grav::instance();`

Reply:
I'll fix this.